### PR TITLE
Clarifies pushing changes to GitHub when adding a page

### PR DESCRIPTION
**Pull request by ns-rse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,11 @@ biography of the presenter(s). To create a new post follow the steps below.
 9. Preview your changes locally to make sure all fields, links etc. render correctly (use `quarto preview` or the
    "Render" button in RStudio).
 
-10. Once everything has been checked and is correct save all changes, commit them to git and push the changes to GitHub.
+10. Once everything has been checked and is correct save all changes, commit them to git and push the changes to
+    GitHub. **NB** There is no need to run `quarto publish gh-pages` locally nor to commit the locally rendered HTML
+    (located under `_site/_` directory and which is ignored via `.gitignore`). The GitHub Workflow (under
+    `.github/workflows/publish.yaml`) handles rendering and publishing the site on GitHub.
+    the pages
```
Comment (tdjames1) [inline README.md]: All good except a couple of stray words on line 45?
```suggestion

```

**Review by tdjames1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ns-rse) [reply README.md]: Good spot, thanks @tdjames1.

**Review by tdjames1 — APPROVED** [review]
Looks good to go.

**Comment by ns-rse** [comment]
Thanks @tdjames1

**Pull request merged**